### ISSUE TITLE
Disk Hardware Details move to Disks

### ISSUE THREAD

**Issue by Movingtarget818** [wh1te909/tacticalrmm#684]
Would love to have the disk information moved from "Hardware Details" to "Disks"

Handy when you have multiple drives located on the one device

See attachment.

![diskdetails](https://user-images.githubusercontent.com/89956127/131775854-82ff5159-afb5-4db7-81ce-45540456f2bc.png)

**Comment by dinger1986** [comment]
I don't agree but up to the devs.

Disks are hardware and should be listed under that. The other side is for disk space. Separate things and more intuitive keeping them separate.

Up to the devs if they want to change it however.

**Comment by johnnyq** [comment]
I agree with @dinger1986 because for example one disk can have many partitions. I think a good approach to this would be to flip it so manufacturer is on top and the drive partitions / letters below it.

**Comment by silversword411** [comment]
I think you'd get a hairball either way. 

What about mirrored software arrays, or Storage Spaces. There's no consistency between disks and partitions.

**Comment by Movingtarget818** [comment]
>  I think a good approach to this would be to flip it so manufacturer is on top and the drive partitions / letters below it.

That could work, i didn't think of the whole "partition" side of it, but having a way of seeing what drives are associated with what disk (without going digging) would be handy
